src/commands/scmt/multilang/enable.ts

```diff
@@ -0,0 +1,67 @@
+/*
+ * Filename: /Users/edahl/Documents/GitHub/sfdx-plugin-scmt/src/commands/scmt/multilang/enable.ts
+ * Path: /Users/edahl/Documents/GitHub/sfdx-plugin-scmt
+ * Created Date: Monday, June 18th 2018, 1:32:26 pm
+ * Author: edahl
+ *
+ * Copyright (c) 2018 Your Company
+ */
+
+
+import { flags } from '@oclif/command'
+import { core } from '@salesforce/command'
+import CommandBase, { PropertyType } from '../../../command_base'
+
+// Initialize Messages with the current plugin directory
+core.Messages.importMessagesDirectory(__dirname);
+
+// Load the specific messages for this file. Messages from @salesforce/command, @salesforce/core,
+// or any library that is using the messages framework can also be loaded this way.
+const messages = core.Messages.loadMessages('@desklabs/scmt', 'multilang');
+
+export default class Enable extends CommandBase {
+  public static description = messages.getMessage('enableCommandDescription');
+  public static examples = [
+    `$ sfdx scmt:multilang:enable`,
+    `$ sfdx scmt:multilang:enable --targetusername myOrg@example.com`
+  ];
+
+  protected static requiresUsername = true;
+
+  public async run(): Promise<any> { // tslint:disable-line:no-any
+    let outputString = messages.getMessage('enableCommandDescription');
+    //navigate to KB settings edit page
+    await this.page.goto(this.buildUrl('/_ui/support/knowledge/KnowledgeSettingsUI/e'));
+    //check for multi
+    let multiCheckbox = await this.page.$('#enableMultilingual1');
+    if (multiCheckbox) {
+      multiCheckbox.click();
+    }
+    //is de selected already? if not, select de and click it
+    let germanSelected = await this.page.$('input[value=de]');
+    if (!germanSelected) {
+      await this.page.evaluate(`
+        document.querySelector("#las_-1 [value=de]").selected = true;
+        document.querySelector('#las_-1').dispatchEvent(new Event('change'));
+      `);
+
+    }
+
+    //figure out row number and build string for the active checkbox
+    let rownum = await this.page.evaluate(`document.querySelectorAll("#langtbl tr").length - 3`);
```

Comment:
Is German always `tr length - 3`?

Reply:
german is not always, the new selection is always tr - 3.  We made an org, added dutch, then ran the command and it worked.

Reply:
you could change `de` in lines 41 & 44 to `it` and things still work as expected. 

@isles4life and I were thinking a good update to this would be to add a parameter for the CLI that is the language to enable

Reply:
![image](https://user-images.githubusercontent.com/15963368/41745685-a8cdc288-755c-11e8-8771-15b703fb7fae.png)

This code does not always select German, but rather the last row in the list.  I recommend pushing this code and adding a continuation issue to resolve later.

Reply:
Ah, did you have german added, but not enabled?

Reply:
@kylehaver - did it work on a fresh org for you?

Reply:
@Skeyelab it works in a fresh org, and that's why we recommend not to change it immediately but create a low priority issue to keep it on the books.